packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -122,10 +127,30 @@ const CreateANewOrganizationFormChild = ({ session }: { session: Ensure<SessionC
         id="createOrg"
         handleSubmit={async (v) => {
           if (!needToCreateOnboarding) {
+            // Resuming existing onboarding - just navigate to next step
             router.push("/settings/organizations/new/about");
-          } else if (!intentToCreateOrgMutation.isPending) {
-            setServerErrorMessage(null);
-            intentToCreateOrgMutation.mutate({ ...v, creationSource: CreationSource.WEBAPP });
+          } else {
+            // Check if this is admin handover flow based on the submitted form value
+            const isAdminHandoverFlow = isAdmin && v.orgOwnerEmail !== session.data.user.email;
+
+            if (isAdminHandoverFlow) {
+              // Admin creating for someone else - submit immediately with just Step 1 data
+              if (!intentToCreateOrgMutation.isPending) {
+                setServerErrorMessage(null);
+                intentToCreateOrgMutation.mutate({ ...v, creationSource: CreationSource.WEBAPP });
+              }
+            } else {
+              // Regular user or admin creating for self - store locally and continue
+              reset({
+                billingPeriod: v.billingPeriod,
+                pricePerSeat: v.pricePerSeat,
+                seats: v.seats,
+                orgOwnerEmail: v.orgOwnerEmail,
+                name: v.name,
+                slug: v.slug,
+              });
+              router.push("/settings/organizations/new/about");
```

Comment:
Determin which flow to trigger base on what role the user is and their target email.

Since we now dont create the intent to the last step (unless admin which we fire here) we dont have an onboardingId